scripts/release/release.js

```diff
@@ -3,7 +3,7 @@ const shell = require('shelljs');
 
 const releaseType = process.argv[2] || 'patch';
 
-if (shell.exec(`npm run compile && npm run lint && npm version ${releaseType} -m "Released version %s" && npm publish && git push`).code !== 0) {
+if (shell.exec(`npm version ${releaseType} -m "Released version %s" && npm publish && git push`).code !== 0) {
```

Comment:
How come this was removed?

Reply:
I didn't do that... that may have been a merge artifact. I can reset it to match master